randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -867,20 +867,28 @@ Extra - room_id: [25]
       Any of the following:
           Can Use Power Bombs
           All of the following:
-              Can Use Bombs
+              Any of the following:
```

Comment:
Dont we have a template for this?

Reply:
we do. i assume schwartz didnt want to use it because it would duplicate the pb requirement, but IMO it should still be used.